com.sap.cloud.lm.sl.cf.process/src/test/java/com/sap/cloud/lm/sl/cf/process/steps/ProcessMtaArchiveStepTest.java

```diff
@@ -127,18 +155,68 @@ private void testDependencies() throws Exception {
 
     }
 
+    private String getFileContent(String fileName) throws IOException {
+        File file = new File("temp");

```

Comment:
Do we really needs new "temp" directory?
Can we process the file directly in the current working dir?